src/benchmarklib/synthetic_table_generator.hpp

```diff
@@ -125,6 +128,8 @@ class SyntheticTableGenerator {
       }
 
       return result;
+    } else {
+      Fail("Requested type not supported.");
```

Comment:
What is this needed for? If you didn't have the else path and would try to initialize it with a type not covered above, the compiler would complain about a missing return value.

Reply:
It would try to convert to a pmr_string. I thought being more explicit about the types here cannot be wrong.